SendWelcomeMessagesActivity/index.ts

```diff
@@ -104,19 +119,20 @@ Per approfondimenti ti invitiamo a consultare la sezione [servizi](ioit://SERVIC
 /**
  * Send a single welcome message using the
  * Digital Citizenship Notification API (REST).
- *
- *  TODO: use italia-commons client with retries
  */
 async function sendWelcomeMessage(
   url: string,
   apiKey: string,
   newMessage: NewMessage
 ): Promise<number> {
-  const response = await request("POST", url)
-    .set("Content-Type", "application/json")
-    .set("Ocp-Apim-Subscription-Key", apiKey)
-    .timeout(DEFAULT_REQUEST_TIMEOUT_MS)
-    .send(newMessage);
+  const response = await timeoutFetch(url, {
```

Comment:
lgtm ! FYI this is not the most problematic call but this one here: https://github.com/pagopa/io-functions-commons/blob/master/src/utils/mailup.ts#L114